src/Resources/views/components/images/image.html.twig

```diff
@@ -1,5 +1,36 @@
 {#
-    This Zicht component easily adds lazyloading and responsive image support in 1 file include. 
+    This Zicht component easily adds lazyloading and responsive image support in a few simple steps.
+
+    TLDR:
+    1. npm install lazysizes --save
+    2. import 'lazysizes'; // in main.ts
+    3. @import "../../vendor/zicht/htmldev-bundle/src/Zicht/Bundle/HtmldevBundle/Resources/sass/components/lazyload.scss"; // in main.scss
```

Comment:
Het is tegenwoordig overigens:
```suggestion
    3. @import "../../vendor/zicht/htmldev-bundle/src/Resources/sass/components/lazyload.scss"; // in main.scss
```

Misschien ook  even op regel 65 :point_down: aanpassen (waar je dit vandaan hebt gekopieerd).